cli/command/cli.go

```diff
@@ -209,12 +209,13 @@ func (cli *DockerCli) Initialize(opts *cliflags.ClientOptions, ops ...Initialize
 
 	cli.configFile = cliconfig.LoadDefaultConfigFile(cli.err)
 
+	cli.contextStore = store.New(cliconfig.ContextStoreDir(), cli.contextStoreConfig)
```

Comment:
Not for this PR, but wondering if we should consider having a `WithContextStore()` ops